CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 ### Fixed
 
+### Added
+- Added `Engine.getWorldBounds`
```

Comment:
Let's add a little more description around this feature on this line, also add the issue number to the end of this line as well `([#729](https://github.com/excaliburjs/Excalibur/issues/729))`